teuthology/provision/pelagos.py

```diff
@@ -0,0 +1,156 @@
+
+import logging
+import requests
+import time
+from teuthology.util.compat import  HTTPError
+import re
```

Comment:
And this line after:
``import requests```

Reply:
we should have some automatic linter for things like that.

Reply:
fixed